hbw/app/models/hbw/fields/select.rb

```diff
@@ -71,7 +71,7 @@ def options_condition
       end
 
       def placeholder
-        definition.fetch('placeholder', '')
+        definition.fetch('placeholder', nil)
```

Comment:
if definition is a hash, this makes no sense, just use `[]` `definition['placeholder']`